newdle/client/src/selectors.js

```diff
@@ -23,3 +23,10 @@ export const shouldConfirmAbortCreation = createSelector(
   areParticipantsDefined,
   (timeslots, hasParticipants) => Object.keys(timeslots).length || hasParticipants
 );
+export const getTitle = state => state.title;
+export const getFullTimeslots = state =>
+  [].concat(
+    ...Object.keys(state.timeslots).map(date =>
+      state.timeslots[date].map(time => `${date}T${time}`)
+    )
+  );
```

Comment:
this can be simplified:

```js
export const getFullTimeslots = state =>
  [].concat(
    ...Object.entries(state.timeslots).map(([date, slots]) =>
      slots.map(slot => `${date}T${slot}`)
    )
  );
```